service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/get.hello.world.file.rpt

```diff
@@ -0,0 +1,6 @@
+connect http://localhost:8700/index.html
```

Comment:
This seems the same as get.index.check.whole.response.rpt so it can be removed, right?